backend/src/services/webhook/webhook-service.ts

```diff
@@ -92,7 +93,8 @@ export const webhookServiceFactory = ({
       actorId,
       projectId: webhook.projectId,
       actorAuthMethod,
-      actorOrgId
+      actorOrgId,
+      actionProjectType: ActionProjectType.Any
```

Comment:
same for these ones here

Reply:
Webhooks are now only for secret-manager, but there is no reason to stop using it for other products as well